src/components/views/settings/KeyBackupPanel.js

```diff
@@ -127,7 +127,7 @@ export default class KeyBackupPanel extends React.PureComponent {
         Modal.createTrackedDialogAsync('Key Backup', 'Key Backup',
             import('../../../async-components/views/dialogs/keybackup/CreateKeyBackupDialog'),
             {
-                secureSecretStorage: SettingsStore.isFeatureEnabled("feature_cross_signing"),
+                secureSecretStorage: false,
```

Comment:
this seems unrelated, but I assume it's fine and expected

Reply:
Actually, pretty sure it's Dave's testing code he did not intend to commit... 😆

Reply:
gah, thanks :/